internal/pkg/daemonsetnotsatisfied/runbook.go

```diff
@@ -8,21 +8,30 @@ import (
 	"k8s.io/client-go/kubernetes"
 
 	"github.com/giantswarm/runbook/pkg/problem"
-	runbookconfig "github.com/giantswarm/runbook/pkg/runbook/config"
 )
 
 const (
-	runbookId        = "daemonset-not-satisfied"
+	RunbookID        = "daemonset-not-satisfied"
```

Comment:
Need this exported, so we can match runbooks by ID when creating them from the outside.